docs/examples/notebooks/hello-world.ipynb

```diff
@@ -120,7 +128,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Observed CL_s: 0.05290116224852556\n",
+      "Observed CL_s: 0.052515541856109765\n",
```

Comment:
this seems like a comparatively big change of 7e-3.. do we have any indication where this comes from?

Reply:
@lukasheinrich I agree, but that actually isn't the result of this PR. If you launch into the `hello-world` notebook on `master`: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/scikit-hep/pyhf/master?filepath=docs%2Fexamples%2Fnotebooks%2Fhello-world.ipynb)

we can see that this behavior is reproducible there given that `qtilde_True` isn't being used in the original example

![qtilde_difference](https://user-images.githubusercontent.com/5142394/89589281-f468c180-d80a-11ea-8c66-69be0cd097ac.png)

I think this was introduced in PR #1005 as we can see comparable differences in the docstring examples: https://github.com/scikit-hep/pyhf/pull/1005/files#diff-9f6c3ebe7e5f12b3657aaa7b3949eaf4R80